docs/How_to_see_nonmatching_code_sections.md

```diff
@@ -0,0 +1,32 @@
+There are many, *many* reasons why what you have decompiled might not match with the retail ROM, ranging from wrong order of statements, to too many/too few variables and parameters, to using the wrong way to do a thing in your code.
+
+Documenting every reason your code might not match is out of the scope of this write-up, but there are ways to figure out exactly which part of your matching-decompilation went wrong. Which in turn can help you fix it.
+
+### Decomp.me
+
+Decomp.me is a 3rd party website that allows decompilers to throw an assembly function inside, set a compiler, and then try to write source code that matches said assembly function.
+
+1. Go to Decomp.me, click "Start Decomping" to open a "New Scratch" window
+2. Once in that window, set the platform to "Nintendo DS (ARMv4T)", and the Compiler preset to "Pokémon HeartGold/SoulSilver"
```

Comment:
Shouldn't it be ARMv5TE here

Reply:
decomp.me currently has it listed as ARMv4T